source/documentation/deploying_services/mysql.md

```diff
@@ -345,6 +345,10 @@ Refer to the [Amazon RDS documentation on read replicas](https://aws.amazon.com/
 
 GOV.UK PaaS does not currently support read replicas, but if you think you would find them useful, please contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk), providing details of your use case.
 
+#### High IOPS plans - MySQL
+
+For services which require heavy disk throughput, GOV.UK PaaS has `high-iops` variants of its MySQL service plans available.
```

Comment:
Will it be clear to the user what we mean by 'heavy disk throughput'?

Reply:
I don't know, but I didn't want to use the term that AWS use to describe "provisioned IO" storage, which is

> For a production application that requires fast and consistent I/O performance

because that might imply that we use "provisioned IO" for this feature, and we don't, we use a hack with gp2 storage instead, (which doesn't necessarily give the same consistency guarantees).

Anyone got any better suggestions?

Reply:
I think that'll very comprehensible even to novice users :ok_hand:

Reply:
What about "services which have a large volume of database reads and writes"?